docs/pages/versions/unversioned/sdk/glass-effect.mdx

```diff
@@ -163,10 +163,29 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+### `isLiquidGlassAvailable`
+
+The `isLiquidGlassAvailable` function let's you check, if the Liquid Glass effect is available in the compiled application.
+It validates the system and compiler versions, as well as the [`Info.plist`](https://developer.apple.com/documentation/BundleResources/Information-Property-List/UIDesignRequiresCompatibility) settings.
```

Comment:
```suggestion
The `isLiquidGlassAvailable` method allows you to check if the Liquid Glass effect is available in the compiled application. It validates the system and compiler versions and the [**Info.plist**](https://developer.apple.com/documentation/BundleResources/Information-Property-List/UIDesignRequiresCompatibility) settings.
```

- Let's use bold text for the file name as per our writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#file-names-directory-names-file-extensions-as-bold-text
- Also improves readability